proposed/http-middleware/middleware-meta.md

```diff
@@ -235,11 +235,30 @@ and a delegate and must return a response. The middleware may:
 #### Why doesn't middleware use `__invoke`?
 
 Doing so would conflict with existing middleware that implements the double-pass
-approach and may want to implement the middleware interface.
+approach and may want to implement the middleware interface for purposes of
+forwards compatibility with this specification.
 
-In addition, classes that define `__invoke` can be type hinted as `callable`,
-which results in less strict typing. This is generally undesirable, especially
-when the `__invoke` method uses strict typing.
+In addition, classes that define `__invoke` can be more loosely type hinted as
+`callable`, which results in less strict typing. This is generally undesirable,
+especially when the `__invoke` method uses strict typing.
```

Comment:
I personally still strongly disagree on this and IMO https://github.com/php/php-src/pull/2265 proves, that this does not align with PHP, its type-system and OCP – But, well, that's should be discussed anywhere else…